config/mail.php

```diff
@@ -237,7 +237,7 @@
     */
 
     'from' => [
-        'address' => env('MAIL_FROM_ADDR', null),
+        'address' => env('MAIL_FROM_ADDR', env('MAIL_ENV_FROM_ADDR')),
```

Comment:
Let's keep this as it was before. We talked this through and `MAIL_ENV_FROM_ADDR` wouldn't be available to us to use here.

```suggestion
        'address' => env('MAIL_FROM_ADDR', null),
```